docs/setup/minikube/getting-started-minikube.md

```diff
@@ -67,32 +59,34 @@ At this stage you should have a functioning Kubernetes cluster, a Docker registr
 
 ```sh
-kubectl create namespace nuclio
+minikube kubectl -- create namespace nuclio
 ```
 
-**Create the RBAC roles** that are required for using Nuclio (provided you didn't disable RBAC when [preparing Minikube](#prepare-minikube)):
-> **Note:** As indicated in the [Minikube preparation](#prepare-minikube) instructions, you are encouraged to look at the [**nuclio-rbac.yaml**](https://github.com/nuclio/nuclio/blob/master/hack/k8s/resources/nuclio-rbac.yaml) file that's used in the following command before applying it.
-
+**Add nuclio to helm repo charts:** the following commands add Nuclio repo charts to your helm repos:
 ```sh
-kubectl apply -f https://raw.githubusercontent.com/nuclio/nuclio/master/hack/k8s/resources/nuclio-rbac.yaml
+helm repo add nuclio https://nuclio.github.io/nuclio/charts
 ```
 
-**Deploy Nuclio to the cluster:** the following command deploys the Nuclio controller and dashboard, among other resources:
+**Deploy Nuclio to the cluster:** the following command deploys Nuclio and its minimum required kubernetes resources (including RBAC roles):
 
 ```sh
-kubectl apply -f https://raw.githubusercontent.com/nuclio/nuclio/master/hack/k8s/resources/nuclio.yaml
+helm --namespace nuclio install nuclio nuclio/nuclio
 ```
 
-Use the command `kubectl get pods --namespace nuclio` to verify both the controller and dashboard are running.
+Use the command `minikube kubectl -- get pods --namespace nuclio` to verify both the controller and dashboard are running.
 
 **Forward the Nuclio dashboard port:** the Nuclio dashboard publishes a service at port 8070. To use the dashboard, you first need to forward this port to your local IP address:
 ```sh
-kubectl port-forward -n nuclio $(kubectl get pods -n nuclio -l nuclio.io/app=dashboard -o jsonpath='{.items[0].metadata.name}') 8070:8070
+minikube kubectl -- port-forward -n nuclio $(kubectl get pods -n nuclio -l nuclio.io/app=dashboard -o jsonpath='{.items[0].metadata.name}') 8070:8070
 ```
 
 ## Deploy a function with the Nuclio dashboard
 
-Browse to `http://localhost:8070` (after having forwarded this port as part of the Nuclio installation). You should see the [Nuclio dashboard](/README.md#dashboard) UI. Choose one of the built-in examples and click **Deploy**. The first build will populate the local Docker cache with base images and other files, so it might take a while, depending on your network. When the function deployment is completed, you can click **Invoke** to invoke the function with a body.
+Browse to `http://localhost:8070` (after having forwarded this port as part of the Nuclio installation).
+You should see the [Nuclio dashboard](/README.md#dashboard) UI.
+Choose one of the built-in examples and click **Deploy**.
+The first build will populate the local Docker cache with base images and other files, so it might take a while, depending on your network.
+When the function deployment is completed, you can click **Invoke** to invoke the function with a body.
```

Comment:
it is not accurate, the current dashboard shows a "Test" button that invokes the function with a body.
so the change should be 
`you can click on **Test** to invoke the function ..`

Reply:
> 
> 
> it is not accurate, the current dashboard shows a "Test" button that invokes the function with a body.
> so the change should be
> `you can click on **Test** to invoke the function ..`
Changing "Invoke" to "Test" isn't related to my comment. I still think we should use the "select" terminology rather than "click". So, if the button caption in the UI is "Test", changing it to this:
> When the function deployment is completed, you can select **Test** to invoke the function with a body.
(Although, why would a function-invocation option be named "Test" ...?)

Reply:
> 
> 
> it is not accurate, the current dashboard shows a "Test" button that invokes the function with a body.
> so the change should be
> `you can click on **Test** to invoke the function ..`
Changing "Invoke" to "Test" isn't related to my comment. I still think we should use the "select" terminology rather than "click". So, if the button caption in the UI is "Test", changing it to this:
> When the function deployment is completed, you can select **Test** to invoke the function with a body.
(Although, why would a function-invocation option be named "Test" ...?)

Reply:
😕 select is confusing as people relate _select_ with HTML select option rather than the _test button_

Reply:
We've been using "Select" throughout all our docs since I came aboard and it was also the style guideline when I worked at IBM, and I've yet to encounter a single report of confusion. I think the intent is clear from the context in this case as well. => I would change it to "select", but if you're adamant about using "click" in this instance, I won't fight you on this any further.